apps/showcase/scripts/update-screenshots/readme.md

```diff
@@ -0,0 +1,14 @@
+# How to generate screenshots for the visual testing
```

Comment:
Maybe you can refer it in the `contributing.md` ?

Reply:
done